src/qt/bitcoingui.cpp

```diff
@@ -925,7 +925,7 @@ void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, double nVer
 {
 // Disabling macOS App Nap on initial sync, disk and reindex operations.
 #ifdef Q_OS_MAC
-    (m_node.isInitialBlockDownload() || m_node.getReindex() || m_node.getImporting()) ? m_app_nap_inhibitor->disableAppNap() : m_app_nap_inhibitor->enableAppNap();
+    m_app_nap_inhibitor->setAppNapEnabled(!m_node.isInitialBlockDownload());
```

Comment:
Note that `isInitialBlockDownload()` already checks `getReindex` and `getImporting`.

Reply:
Also, say goodbye to this ternary.